windows-driver-docs-pr/stream/frame-server-custom-media-source.md

```diff
@@ -168,9 +169,9 @@ HKR,,CustomCaptureSourceClsid,,%CustomCaptureSource.CLSID%
 HKR,,FriendlyName,,%CustomCaptureSource.Desc%
 
 [CustomCaptureSource.ComRegistration]
-HKCR,CLSID\%CustomCaptureSource.CLSID%,,,%CustomCaptureSource.Desc%
-HKCR,CLSID\%CustomCaptureSource.CLSID%\InprocServer32,,%REG_EXPAND_SZ%,%CustomCaptureSource.Location%
-HKCR,CLSID\%CustomCaptureSource.CLSID%\InprocServer32,ThreadingModel,,Both
+HKR,Classes\CLSID\%CustomCaptureSource.CLSID%,,,%CustomCaptureSource.Desc%
```

Comment:
Doesn't the camera stack not support device relative COM registrations until a build much later than the 22000 this INF is targeted at?

Reply:
You are correct. 25326 should be the minimum